purchases/src/androidTestIntegrationTest/kotlin/com/revenuecat/purchases/BasePurchasesIntegrationTest.kt

```diff
@@ -40,7 +40,7 @@ open class BasePurchasesIntegrationTest {
     protected open val initialActivePurchasesToUse: Map<String, StoreTransaction> = emptyMap()
     protected open val initialForceServerErrors: Boolean = false
 
-    protected val testTimeout = 7.seconds
+    protected val testTimeout = 15.seconds
```

Comment:
Right, in Android, we are also explicitly asking for the mapping to , but the delay is still added (note that that's not a public API, so users won't hold waiting for that data).

We could try to mock `Delay`, but I'm concerned that if we do that, the integration tests won't reflect an actual production behavior... I guess we could decrease the jittering a lot though, to maybe a couple seconds tops, so we don't have to wait so long for these tests. Will try to do that.

Reply:
Mocking wasn't really easy since that class is very internal and it would have increased the complexity of the integration tests a ton. I decided to just separate the jittering values for the integration test flavor that already exists: https://github.com/RevenueCat/purchases-android/pull/1017/commits/c8204da23faf7955277647470550bedf1abcc709.

I also considered creating a flag when running integration tests but those mostly require using reflection or creating build types for tests and would make it so the test code is also present in production code. Also, I think it's not worth it as long as it's just modifying some constants. If we end up having to modify the logic in multiple places for integration tests (we hopefully shouldn't), then I think we can consider adding the flag instead.